### PR TITLE
Update rejected petitions list view

### PR DESCRIPTION
**Pull request by otlaitil**
* Changed default sort ordering
* Disabled showing signature count
* Updated view to match new designs

### CODE REVIEW

**Comment by otlaitil** [comment]
Addressed @h-lame's PR comments.

File: app/assets/stylesheets/petitions/views/_shared.scss [hunk 1]
```diff
@@ -91,6 +91,10 @@
   }
 }
 
+.petition-item.petition-rejected {
```
Comment (alexander-bobin) [inline app/assets/stylesheets/petitions/views/_shared.scss]: Nice variation combo. Leave it as is but in future, a variation should have the base in the name so something like `.petition-item-rejected` would be better

**Comment by alexander-bobin** [comment]
LGTM :+1:

**Comment by davidbasalla** [comment]
:shipit:

**Pull request merged**